### PR TITLE
Add details to MeteoSwiss events

### PR DESCRIPTION
**Pull request by frozenhelium**

### CODE REVIEW

File: src/components/domain/RiskImminentEvents/MeteoSwiss/EventListItem/index.tsx [hunk 1]
```diff
@@ -19,16 +19,20 @@ function EventListItem(props: Props) {
...
-            hazard_name,
+            hazard_type_display,
+            country_details,
             start_date,
+            hazard_name,
         },
         onExpandClick,
     } = props;
 
+    const hazardName = `${hazard_type_display} - ${country_details?.name ?? hazard_name}`;
```
Comment (tnagorra) [inline src/components/domain/RiskImminentEvents/MeteoSwiss/EventListItem/index.tsx:30]: Is this fallback correct?

File: src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx [hunk 2]
```diff
@@ -1,53 +1,195 @@
...
         pending,
-        exposure,
     } = props;
 
-    // eslint-disable-next-line no-console
-    console.info(exposure);
+    const strings = useTranslation(i18n);
+
+    // FIXME: use translation?
+    const hazardName = `${hazard_type_display} - ${country_details?.name ?? hazard_name}`;
```
Comment (tnagorra) [inline src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx:46]: Is this correct?

File: src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx [hunk 3]
```diff
@@ -1,53 +1,195 @@
...
+                        contentViewType="vertical"
+                        headingLevel={5}
+                        withHeaderBorder
+                    >
+                        {impactList.map((impact) => (
+                            <TextOutput
+                                label={impact.label}
+                                value={impact.value}
+                                valueType="number"
+                                strongValue
```
Comment (tnagorra) [inline src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx]: key is missing

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/components/domain/RiskImminentEvents/MeteoSwiss/EventListItem/index.tsx (hunk 1 above)
Comment (frozenhelium) [reply src/components/domain/RiskImminentEvents/MeteoSwiss/EventListItem/index.tsx:30]: Yes

File: src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx (hunk 2 above)
Comment (frozenhelium) [reply src/components/domain/RiskImminentEvents/MeteoSwiss/EventDetails/index.tsx:46]: Yes

**Pull request merged**